ui/droppable.js

```diff
@@ -33,6 +33,7 @@ $.widget( "ui.droppable", {
 		accept: "*",
 		activeClass: false,
 		addClasses: true,
+		classes: {},
 		greedy: false,
 		hoverClass: false,
```

Comment:
I think thats the only way that makes sense for this to work through classes. It's also consistent with how the `classes